zipline/finance/blotter.py

```diff
@@ -114,6 +114,9 @@ def order(self, sid, amount, style, order_id=None):
 
         return order.id
 
+    def order_batch(self, orders):
```

Comment:
Do we want to document this in some way? (Or at least comment it so we don't remove it?)